workshops/admin.py

```diff
@@ -11,11 +11,16 @@
     Badge,
 )
 
+
+class RoleAdmin(admin.ModelAdmin):
+    list_display = ('name', 'verbose_name', 'show_in_training_request_form')
```

Comment:
`verbose_name` is a name that is displayed in the application form in `Previous involvement with Software Carpentry or Data Carpentry` question. `name` is internal name that we cannot change.

For most roles, `verbose_name` and `name` are the same, but that's not always the case, i.e. I've introduced `Role(name='contributor', verbose_name='Contributed to lesson materials')`.